src/ray/util/filesystem.cc

```diff
@@ -60,4 +61,18 @@ std::string GetUserTempDir() {
   return result;
 }
 
+std::string CompleteReadFile(const std::string &fname) {
+  std::ifstream file(fname);
+  RAY_CHECK(file.good()) << "Fails to open file " << fname;
+
+  std::ostringstream buffer;
+  buffer << file.rdbuf();
+  RAY_CHECK(file.good()) << "Fails to read from file " << fname;
```

Comment:
After reading, the file should be in the eof state which is not good()?

Reply:
I don't think so.

I did a simply experiment.

Setup:
```
[~] 
ubuntu@hjiang-devbox-pg$ echo "helloworld" > hello.out
[~] 
ubuntu@hjiang-devbox-pg$ g++ -std=c++17 fstream_test.cc -o test
[~] 
ubuntu@hjiang-devbox-pg$ ./test
0
helloworld
```

Code
```
#include <fstream>
#include <sstream>
#include <cassert>
#include <iostream>

int main() {
    std::fstream fs("/home/ubuntu/hello.out");
    std::ostringstream buffer;
    buffer << fs.rdbuf();
    std::cout << fs.rdstate() << std::endl;
    std::cout << std::move(buffer).str() << std::endl;
    assert(fs.good());
    return 0;
}
```